test/module/irohad/ametsuchi/postgres_executor_test.cpp

```diff
@@ -373,20 +349,44 @@ namespace iroha {
       auto cmd_result = execute(*add_asset, true);
 
       std::vector<std::string> query_args{
-          account->accountId(), uint256_halfmax.toStringRepr(), asset_id, "1"};
+          account_id, uint256_halfmax.toStringRepr(), asset_id, "1"};
       CHECK_ERROR_CODE_AND_MESSAGE(cmd_result, 4, query_args);
     }
 
+    struct MockPeer : public shared_model::interface::Peer {
```

Comment:
Should be in different place, probably at test/module/shared_model/interface_mocks.hpp